### PR TITLE
Codewars

### PR DESCRIPTION
**Pull request by ddadivan**
Codewars 24.08

### CODE REVIEW

File: Codewars/25.08.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+}
+
+console.log(hero(10, 5));
+console.log(hero(7, 4));
+
+
+//Write a program that finds the summation of every number from 1 to num.
+// The number will always be a positive integer greater than 0.
+// Your function only needs to return the result, what is shown between parentheses in the example below is how you reach that result and it's not part of it, see the sample tests.
+var summation = function (num) {
```
Comment (YuriiSkyba1) [inline Codewars/25.08.js:153]: why var

File: Codewars/26.08.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+function sum (numbers) {
+    return numbers.length ? numbers.reduce((acc, cur) => acc + cur) : 0
+};
+console.log(sum([1, 5.2, 4, 0, -1]));
+
+
+//In this simple exercise, you will build a program that takes a value, integer , and returns a list of its multiples up to another value, limit .
+// If limit is a multiple of integer, it should be included as well. There will only ever be positive integers passed into the function, not consisting of 0.
+// The limit will always be higher than the base.
+function findMultiples(integer, limit) {
```
Comment (YuriiSkyba1) [inline Codewars/26.08.js:35]: arrow func

File: Codewars/26.08.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+console.log(arrayPlusArray([1, 2, 3], [4, 5, 6]));
+
+
+
+//Write a function that takes an array of words and smashes them together into a sentence and returns the sentence.
+// You can ignore any need to sanitize words or add punctuation, but you should add spaces between each word.
+// Be careful, there shouldn't be a space at the beginning or the end of the sentence!
+function smash (words) {
```
Comment (YuriiSkyba1) [inline Codewars/26.08.js:17]: arrow func

File: Codewars/25.08.js (hunk 1 above)
Comment (ddadivan) [reply Codewars/25.08.js]: fixed

File: Codewars/26.08.js (hunk 2 above)
Comment (ddadivan) [reply Codewars/26.08.js]: fixed

File: Codewars/26.08.js (hunk 3 above)
Comment (ddadivan) [reply Codewars/26.08.js]: fixed

**Review by YuriiSkyba1 — APPROVED** [review]

File: Codewars/25.08.js [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+    return costDays;
+}
+
+console.log(rentalCarCost(3));
+console.log(rentalCarCost(5));
+console.log(rentalCarCost(8));
+
+
+//Write a function that takes a list of strings as an argument and returns a filtered list containing the same elements but with the 'geese' removed.
```
[the remark this answers is not in the document]
Comment (ddadivan) [reply Codewars/25.08.js:96]: fixed

File: Codewars/25.08.js [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+
+console.log(sumStr("4", "5"));
+
+
+
+// To complete this Kata you need to make a function multiplyAll/multiply_all which takes an array of integers as an argument.
+// This function must return another function, which takes a single integer as an argument and returns a new array.
+//
+// The returned array should consist of each of the elements from the first array multiplied by the integer.
+
```
[the remark this answers is not in the document]
Comment (ddadivan) [reply Codewars/25.08.js:119]: fixed

**Review by YuriiSkyba1 — APPROVED** [review]

**Pull request merged**